R4ML/R/zzz.R

```diff
@@ -85,286 +71,128 @@ auto_start_session <- function() {
 # call the systemML as well as SparkR functionality as needed
 # if sparkR is already initialized then it wil re-initialize it
 .onLoad <- function(libname, pkgname) {
-  logsrc <- r4ml.env$PACKAGE_NAME
-
-  if ("SparkR" %in% loadedNamespaces()) {
-    if (exists(".sparkRjsc", envir = SparkR:::.sparkREnv)) {
-      r4ml.warn(logsrc, "Reloading SparkR")
-      r4ml.reload.SparkR()
-    } else {
-      r4ml.info(logsrc, "Loading SparkR")
-      r4ml.load.SparkR()
-    }
-  }
+  logsrc <- ".onLoad"
 
-  r4ml.startup.message(libname, pkgname)
-
-  if(auto_start_session()) {
+  if (Sys.getenv("R4ML_AUTO_START") == "1") {
     r4ml.info(logsrc, "Auto starting session")
     r4ml.session()
   }
 }
 
-r4ml.startup.message <- function(libname, pkgname, ...) {
-  logsrc <- r4ml.env$PACKAGE_NAME
-  # print the logo
+.onAttach <- function(libname, pkgname) {
   packageStartupMessage(r4ml_ascii())
 
   # print the version
-  if(nchar(libname) > 0 & nchar(pkgname) > 0) { 
+  if (nchar(libname) > 0 & nchar(pkgname) > 0) { 
     desc <- utils::packageDescription(pkg = pkgname, lib.loc = libname)
-    vstr <- sprintf("[%s]: version %s", logsrc, desc$Version)
-    packageStartupMessage(vstr)
-  }
-}
-
-r4ml.post.startup.message <- function(...) {
```

Comment:
why this removed? we need this placeholder function post.startup for future changes if any